packages/policy/src/export.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {BigQuery, BigQueryDate} from '@google-cloud/bigquery';
+import {PolicyResult} from './policy';
+
+const bigquery = new BigQuery();
+const datasetId = 'PolicyResults';
+const tableId = 'PolicyResults';
+
+interface BigQueryPolicyResult extends PolicyResult {
+  recordDate: BigQueryDate;
+}
+
+/**
+ * Given a single policy result, write the record to a BigQuery table.
+ * @param result The Policy Result for a single repository
+ */
+export async function exportToBigQuery(result: PolicyResult) {
+  const r = result as BigQueryPolicyResult;
+  r.recordDate = BigQuery.date(new Date().toISOString().slice(0, 10));
```

Comment:
I think this would be worth a comment, and we should test that running the job twice in the same 24 hours doesn't fail -- so that we can re-run to pull in updated data.

Reply:
Yeah I don't think this was needed, it was historical cruft.